frontend/src/metrics.rs

```diff
@@ -24,19 +24,19 @@ use std::{
     time::Duration,
 };
 
-use anyhow::Result;
-use log::*;
-use parking_lot::Mutex;
```

Comment:
It would b e better to just add once_cell here and not move the block.

Reply:
Thanks for your help, so essentially I should substitute the place of `lazy_static` import with the `once_cell` directly, everywhere no need to reorder (Iam I able to get it correct ?)